api/v2/app/auth/helpers.py

```diff
@@ -0,0 +1,72 @@
+import re
+from werkzeug.security import generate_password_hash, check_password_hash
+from functools import wraps
+from flask import request, jsonify, make_response
+
+from app.models import User
+
+
+def email_validator(email):
+    '''validates user provided email'''
```

Comment:
For consistency please use triple double quotes for your docstrings

Reply:
Noted